components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/util/ClaimConstants.java

```diff
@@ -54,6 +54,7 @@ public class ClaimConstants {
     public static final String MAX_LENGTH = "maxLength";
     public static final String IS_SYSTEM_CLAIM = "isSystemClaim";
     public static final String SHARED_PROFILE_VALUE_RESOLVING_METHOD = "SharedProfileValueResolvingMethod";
+    public static final String EXTERNAL_CLAIM_ADDITION_NOT_ALLOWED_FOR_DIALECT = "SkipAddExternalClaim";
```

Comment:
Let's update the value.

Reply:
Fixed with 66ba9e01824399ee9e309f7d59dc004ef5e48aa6